yaas/parsers/service.py

```diff
@@ -0,0 +1,23 @@
+# coding: utf-8
+
+from __future__ import absolute_import
+from __future__ import print_function
+
+import inspect
+
+from .. import service
+from .. import config
+from .. import utils
+
+def define_subcommand(parent_parser):
```

Comment:
I see what you're saying, and I like it. The API you would be looking for is:

blueprint::define_command():
  //make parser
  return parser

__main__::cli_args():
  //make parser
  parser.add_subparser(blueprint.define_command())

Unfortunately argparse parsers must be defined sequentially--top to bottom.